config/postgres/initdb/scripts/create-tables.sql

```diff
@@ -0,0 +1,58 @@
+\c record_expunge;
+
+CREATE EXTENSION "uuid-ossp";
+CREATE EXTENSION citext;
+
+CREATE TABLE users (
+    user_id         UUID NOT NULL,
+    username        CITEXT UNIQUE NOT NULL,
+    email           CITEXT UNIQUE NOT NULL,
+    hashed_password TEXT NOT NULL,
+    password_salt   TEXT NOT NULL,
+    admin           BOOLEAN NOT NULL DEFAULT FALSE,
+    date_created    TIMESTAMP WITH TIME ZONE DEFAULT now(),
+    date_modified   TIMESTAMP WITH TIME ZONE DEFAULT now(),
+    PRIMARY KEY (user_id)
+);
+
+
+CREATE TABLE clients (
+    client_id     UUID NOT NULL,
+    first_name    TEXT NOT NULL,
+    last_name     TEXT NOT NULL,
+    date_of_birth TEXT NOT NULL,
+    date_created  TIMESTAMP WITH TIME ZONE DEFAULT now(),
+    date_modified TIMESTAMP WITH TIME ZONE DEFAULT now(),
+    PRIMARY KEY (client_id),
+    UNIQUE (first_name, last_name, date_of_birth)
+);
+
+
+CREATE TABLE result_codes (
+    result_code_id UUID NOT NULL,
+    code           TEXT UNIQUE NOT NULL,
+    PRIMARY KEY (result_code_id)
+);
+
+
+CREATE TABLE rules (
+    rule_id UUID NOT NULL,
+    text    TEXT UNIQUE NOT NULL,
+    PRIMARY KEY (rule_id)
+);
+
+
+CREATE TABLE analysis (
+    client_id UUID NOT NULL REFERENCES clients(client_id) ON DELETE CASCADE,
+    case_id   TEXT NOT NULL,
+    result_code_id UUID NOT NULL REFERENCES result_codes(result_code_id) ON DELETE CASCADE,
+    statute        TEXT NOT NULL,
+    date_eligible  TIMESTAMP WITH TIME ZONE,
+    rule_ids       UUID[] NOT NULL,
+    date_created   TIMESTAMP WITH TIME ZONE DEFAULT now(),
+    date_modified  TIMESTAMP WITH TIME ZONE DEFAULT now(),
+    expunged       BOOLEAN,
+    date_expunged  TIMESTAMP WITH TIME ZONE,
+    user_id        UUID NOT NULL REFERENCES users(user_id) ON DELETE CASCADE,
+    PRIMARY KEY (client_id, case_id)
+);
```

Comment:
Speaking with Michael it seems like we may need to create separate table for "time eligibility analysis" if it makes sense to do so. The info that would need to go on this would be
- if "10 year rule" passed - means all charges are expungeable
- if most recent conviction is eligible, etc.
- date of eligibility for charges other than MRC.
...

These would be listed on the UI page at the top. @hmarcks 

So we either need to add those info in this table or create a new one.